object.go

```diff
@@ -281,22 +281,60 @@ func (js *js) DeleteObjectStore(bucket string) error {
 	return js.DeleteStream(stream)
 }
 
-func sanitizeName(name string) string {
-	stream := strings.ReplaceAll(name, ".", "_")
-	return strings.ReplaceAll(stream, " ", "_")
+func encodeName(name string) string {
+	return base64.URLEncoding.EncodeToString([]byte(name));
+}
+
+// toDefaultMetaSubject and toPublishMetaSubject will diverge once there is account support
+func toDefaultMetaSubject(obs *obs, name string) string {
+	return fmt.Sprintf(objMetaPreTmpl, obs.name, encodeName(name))
```

Comment:
This looks same as func below. Also if 1 liners why not inline?

Reply:
right now it is the same as the func below. To handle cross account we need to handle subjects differently in certain spots. These are just placeholders for when that work gets done.

Reply:
removed / re-inlined with comments